src/components/KMultiselect/KMultiselect.vue

```diff
@@ -81,135 +104,109 @@
                 </template>
               </KBadge>
               <KBadge
-                v-if="!expandSelected && invisibleSelectedItems.length"
+                v-if="invisibleSelectedItems.length"
+                :appearance="getBadgeAppearance()"
```

Comment:
I wasn't sure where else to put this comment: I have concerns on the `hiddenItemsTooltip` - what if there are 15 hidden items? Do we really want to display them all in a tooltip?